### PR TITLE
Add new Turkish stories

### PR DESCRIPTION
**Pull request by jayhoon**
@casey-collab
Please review and sign off on my additions to the story

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: turkce/darbe-kumpas/darbeler.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Su siteden butun darbeleri desifre edebilirsin ozellikle paralel olanalarini
+[star-darbeler](www.star.com.tr)
```
Comment (casey-collab) [inline turkce/darbe-kumpas/darbeler.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: turkce/hatmisekeri.md [hunk 2]
```diff
@@ -17,3 +17,4 @@ Bu durumda ne yapardiniz?
 [Duvarları yeme başla?](../turkce/duvarlari-ye/duvarlari-ye.md)
 
 [Duvarları çatalla kaz?](../turkce/catalla-kaz/catalla-kaz.md)
+[Darbe tesebbusu desifre et](../turkce/darbe-kumpas/darbeler.md)
```
Comment (casey-collab) [inline turkce/hatmisekeri.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: turkce/darbe-kumpas/darbeler.md (hunk 1 above)
Comment (cbuckey-uda) [inline turkce/darbe-kumpas/darbeler.md]: I believe you will need to add "http://" at the beginning of the link.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**